app/operations/ops/developer/onboarding_progress.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Ops
+  module Developer
+    # This is class to calculate percent onboarding progress users
+    class OnboardingProgress
+      def initialize(user)
+        @user = user
+      end
+
+      def percent
+        total = @user.test_task_assignments.size || Ops::Developer::Screening::NUMBER_OF_ASSIGNED_TEST_TASKS
```

Comment:
```
def total_steps
  total = @user.test_task_assignments.size || Ops::Developer::Screening::NUMBER_OF_ASSIGNED_TEST_TASKS
  # We add 2 because of slack and github
  total + 2
end
```